src/utils/interfaces.ts

```diff
@@ -37,6 +37,10 @@ export interface userType {
   name: string;
   permissions: Record<string, boolean>;
   profile_img?: string;
+  phone:string;
```

Comment:
this may cause issues in places where it is already being used

we will have known for sure if you weren't bypassing the checks